acceptance_tests/pages/survey.py

```diff
@@ -0,0 +1,20 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to():
+    browser.visit(Config.RESPONSE_OPERATIONS_UI)
```

Comment:
👍 has this been updated in config.py too?

Reply:
Yes it was a duplicate so I removed the duplicate